.gitignore

```diff
@@ -44,4 +44,7 @@ test-results/
 # terraform .tfstate files
 **/.terraform
 *.tfstate*
-*.terraform*
\ No newline at end of file
+*.terraform*
+
+# Finder state files
+.DS_Store
```

Comment:
Shocked that we've had multiple folks on the team using macs (myself included!) for multiple months and none of us had run into these .DS_Store files before.  Problem solved.

Reply:
Thanks for doing this. I've been just mentally ignoring these :)